pkg/client/client_test.go

```diff
@@ -202,7 +205,30 @@ func TestAddEventsEndpointUrlWithTrailingSlash(t *testing.T) {
 	t.Setenv("SCALYR_SERVER", "https://app.scalyr.com/")
 	cfg2, err := config.New(config.FromEnv())
 	assert.Nil(t, err)
-	sc2, err := NewClient(cfg2, nil, zap.Must(zap.NewDevelopment()))
+	sc2, err := NewClient(cfg2, nil, zap.Must(zap.NewDevelopment()), nil)
 	require.Nil(t, err)
 	assert.Equal(t, sc2.addEventsEndpointUrl, "https://app.scalyr.com/api/addEvents")
 }
+
+func TestUserAgent(t *testing.T) {
+	t.Setenv("SCALYR_SERVER", "https://app.scalyr.com/")
+	libraryConsumerUserAgentSuffix := "OtelCollector;0.80.0;traces"
+	numCpu := fmt.Sprint(runtime.NumCPU())
+	cfg, err := config.New(config.FromEnv())
+	assert.Nil(t, err)
+	client, err := NewClient(cfg, nil, zap.Must(zap.NewDevelopment()), &libraryConsumerUserAgentSuffix)
+	clientId := client.Id.String()
+	require.Nil(t, err)
+	assert.Equal(t, client.userAgent, "datasetexporter;"+version.Version+";"+version.ReleasedDate+";"+clientId+";"+runtime.GOOS+";"+runtime.GOARCH+";"+numCpu+";"+libraryConsumerUserAgentSuffix)
```

Comment:
Do you happen to have a complete string somewhere? Just want to see what kind of format `version.ReleaseDate` uses and how the complete string looks like.

Reply:
eg `datasetexporter;0.0.8;2023-05-30;8622d8b4-0f5b-44d3-9160-52581a13b43e;darwin;arm64;10;OtelCollector;0.80.0;traces`